requirements.txt

```diff
@@ -10,17 +10,17 @@ lxml==4.4.1
 # Putting upgrade on hold due to v1.0.0 using sha512 instead of sha1 by default
 itsdangerous==0.24  # pyup: <1.0.0
 
-git+https://github.com/alphagov/digitalmarketplace-utils.git@50.0.1#egg=digitalmarketplace-utils==50.0.1
+git+https://github.com/alphagov/digitalmarketplace-utils.git@ldeb-add-forms-errors-helpers#egg=digitalmarketplace-utils==9999
```

Comment:
This is merged now, right?

Reply:
Yeah

Reply:
I've pushed a version with updated requirements